config.go

```diff
@@ -386,6 +383,16 @@ func loadConfig() (*config, error) {
 		return nil, err
 	}
 
+	// At least one RPCListener is required
```

Comment:
Minor nit: missing period at the [end of sentences on the comments](https://github.com/lightningnetwork/lnd/blob/master/docs/code_contribution_guidelines.md#43-code-documentation-and-commenting).